spec/rails_helper.rb

```diff
@@ -3,6 +3,8 @@
 
 require "spec_helper"
 ENV["RAILS_ENV"] ||= "test"
+# Lockbox master key is specific to test environment
+ENV["LOCKBOX_MASTER_KEY"] = "5b1f54d9c713b028b34a34b2bf95b4e22150fcd636069efc67cf7bff64ddfb04"
```

Comment:
Overriding env variables gets a bit faffy - I reckon this would be better off in `config/environments/test.rb` as

```ruby
Lockbox.master_key = 'blahdiblah'
```